src/sql/parts/query/editor/gridPanel.ts

```diff
@@ -156,6 +156,8 @@ export class GridPanel extends ViewletPanel {
 			tables.push(table);
 		}
 
+		// possible to need a sort?
```

Comment:
Looks like you meant to look into this?

Reply:
No, this is mainly a note for later, current we run the queries sychronously so I'm pretty sure its impossible for the result sets to come out of order. But in the future this may not be the case.